### ISSUE TITLE
[Registrierung] Availabilities erweitern

### ISSUE THREAD

**Issue by hollesse** [innoq/seawatch_planner#101]
In meinen Augen macht es Sinn die Availabilities so zu erweitern, dass ich sagen kann ich kann den kompletten Zeitraum, oder ich kann in diesem Zeitraum beispielsweise 7 Tage oder 5 Werktage. 

Seht ihr das auch so? @janmax @mschwarz82 @nadyes

**Comment by janmax** [comment]
Ja, das sollte sich ja auch leicht im Datenmodel umsetzen lassen.
